skyportal/handlers/api/source.py

```diff
@@ -260,6 +260,7 @@ def get(self, obj_id=None):
             source_info["last_detected"] = s.last_detected
             source_info["gal_lat"] = s.gal_lat_deg
             source_info["gal_lon"] = s.gal_lon_deg
+            source_info["alias"] = ""  # placeholder until we get TNS names
```

Comment:
I was under the impression that this is going to come in very soon. 

@dmitryduev do you know when we can expect to have TNS names? or at least where I can find them in them when they appear?

Reply:
```suggestion
```
Instead of an empty string placeholder, we should simply be able to handle the case where this is null or undefined on the front-end gracefully